spec/controllers/request_controller_spec.rb

```diff
@@ -2939,30 +2939,27 @@ def expect_redirect(status, redirect_path)
 end
 
 describe RequestController, "when caching fragments" do
+  let(:info_request) { FactoryBot.create(:info_request_with_incoming) }
+  let(:incoming_message) { info_request.incoming_messages.first }
+
   it "should not fail with long filenames" do
-    long_name = "blahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblah.txt"
-    info_request = double(InfoRequest, :prominence => 'normal',
-                                       :is_public? => true,
-                                       :embargo => nil)
-    incoming_message = double(IncomingMessage, :info_request => info_request,
-                            :parse_raw_email! => true,
-                            :info_request_id => 132,
-                            :id => 44,
-                            :get_attachments_for_display => nil,
-                            :apply_masks => nil,
-                            :prominence => 'normal',
-                            :is_public? => true)
-    attachment = FactoryBot.build(:body_text, :filename => long_name)
-    allow(IncomingMessage).to receive(:find).with("44").and_return(incoming_message)
-    allow(IncomingMessage).to receive(:get_attachment_by_url_part_number_and_filename).and_return(attachment)
-    allow(InfoRequest).to receive(:find).with("132").and_return(info_request)
-    params = { :file_name => long_name,
-               :controller => "request",
-               :action => "get_attachment_as_html",
-               :id => "132",
-               :incoming_message_id => "44",
-               :part => "2" }
+    long_name = 'blahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblahblah.pdf'
```

Comment:
This line makes my editor slow to a crawl 😭  
```suggestion
    long_name = 'blah' * 150 + '.pdf'
```

Reply:
good suggestion - one sec